src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -1967,5 +2009,5 @@
       "Size": 341228
     }
   },
-  "PackageSize": 7930399
+  "PackageSize": 8701694
 }
```

Comment:
This project seemed to regress in app size a lot?

Reply:
I reverted the `.apkdesc` changes, I think we were getting an old runtime build. Hoping the next build works better.